src/VisualStudio/Core/Def/Implementation/TableDataSource/VisualStudioBaseDiagnosticListTable.LiveTableDataSource.cs

```diff
@@ -152,6 +150,8 @@ private void OnDiagnosticsUpdated(object sender, DiagnosticsUpdatedArgs e)
                         return;
                     }
 
+                    ChangeProgress(e);
```

Comment:
❓ How often is this called?

Reply:
whenever there is change in diagnostics displayed in error list

Reply:
this runs in BG. so I don't think it will affect responsiveness of VS and there are bunch of work after this so, I dont think it will impact perf in user noticeable way.

Reply:
just to check, is there perf markers to know if thing slow down here?

Reply:
no. but since it runs in BG, I dont believe it will cause perf issue. at least something user or RPS can notice.